rtl.css

```diff
@@ -1793,6 +1793,86 @@ blockquote, q {
   .search article .entry-meta .author a {
     padding-right: 0px; }
 
+
+ul {
+  list-style-type: disc; }
+
+.wp-block-latest-posts.aligncenter {
+  margin-right: 3em; }
+
+.wp-block-verse {
+  color: #8f8f8f; }
+
+.wp-block-quote::before, blockquote:before {
+  content: "";
+  border: 0px; }
+
+.wp-block-quote, blockquote, q {
```

Comment:
@PremalathaKumar Again targeting all blockquote? I believe it should be something specific like `posts/entry/article`